third-party/chpl-venv/virtualenv.txt

```diff
@@ -1 +1 @@
-virtualenv==12.0.5
+virtualenv==15.1.0
```

Comment:
Looks like [15.2.0](https://pypi.python.org/pypi/virtualenv) just came out - any reason not to use that?